app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class NotificationMailer < ApplicationMailer
+
```

Comment:
Nice, probably can remove these extra spaces too, wonder if we should pass in the old project code so that we can say "has been changed from X to Y"? Any you could probably get the current user too to say who changed it. Up to you though if you think it's useful...